### DISCUSSION TITLE
(Question) Grafana usage

### DISCUSSION THREAD

**Discussion by alphafox02** [szpajder/dumphfdl#7] [Q&A]
Hello. If there's a better place to ask, please let me know. I've included dumphfdl into DragonOS Focal, a linux distro geared towards SDRs. I noticed your screenshots using Grafana. I've did a quick setup in the past for another project using Grafana, but found it interesting what you were able to do with dumphfdl data.

Would you happen to have any notes or directions I could follow to do the same thing? I'd in turn share the setup. Thank you.

**Comment by szpajder** [comment]
Thank you for your interest in my project! Please be sure to get the latest version 1.1.0 released just yesterday. It has some significant improvements over 1.0.0.

As you are not the first person asking about Grafana integration, maybe I'll just add a section to the README.md with a description of the supported metrics and some ideas on how they might be turned into graphs. I'll ping you when it's done.

**Comment by alphafox02** [comment]
Thank you and latest version from git pulled today. Updated libacars too and received data with an rsp1a and YouLoop perfectly first try. Thank you for such great projects.

**Comment by szpajder** [comment]
I've added docs for StatsD metrics. In order not to clutter the main README, I've put them in [doc/STATSD_METRICS.md](https://github.com/szpajder/dumphfdl/blob/master/doc/STATSD_METRICS.md).
